home/management/commands/updatecourses.py

```diff
@@ -0,0 +1,139 @@
+import re
+import requests
+from datetime import datetime
+
+from django.core.management import BaseCommand
+from argparse import RawTextHelpFormatter
+
+from home.models import Course, Professor, ProfessorCourse, ProfessorAlias
+from home.utils import Semester
+
+class Command(BaseCommand):
+    requires_migrations_checks = True
```

Comment:
I think this is pretty silly to require for the future, since this command doesn't actually rely on arbitrary migrations being up to date in general.